pyvis/templates/template.html

```diff
@@ -1,7 +1,7 @@
 <html>
 <head>
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vis/4.16.1/vis.css" type="text/css" />
-<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/vis/4.16.1/vis-network.min.js"> </script>
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/vis-network@latest/styles/vis-network.css" type="text/css" />
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/vis-network@latest/dist/vis-network.min.js"> </script>
```

Comment:
I'm not sure it's a good idea to put the latest version, because it introduces a chance, that backward compatibility will be broken at some moment.

But if we can set up a CI with 2 different jobs: one to build with some fixed version, one to build with the latest one, it would be much more useful IMO.

Reply:
I can see that, but on the other hand half of the issues in this Repo are from issues with the old version and from People that don't understand what this repos does and that graph rendering is done from [vis-network](https://github.com/visjs/vis-network).